src/resolvers/git.cr

```diff
@@ -113,7 +113,7 @@ module Shards
 
     protected def self.has_git_command?
       if @@has_git_command.nil?
-        @@has_git_command = Process.run("command -v git", shell: true).success?
+        @@has_git_command = (Process.run("git", ["--version"]).success? rescue false)
```

Comment:
```suggestion
        @@has_git_command = Process.run("git", ["--version"]).success? rescue false
```

Reply:
It can work both ways but just consider what advantages we're weighing here (picking one or the other):
* Saving two characters.
* Not having to think what the rescue *really* applies to every time when reading the code. I literally didn't know what the priority would be and I wanted to save everyone the cognitive load and the possible bugs.

Reply:
Ok, I thought it's pretty clear what's going on here (there's nothing else that 'rescue' could apply to), but it's your call.